.babelrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can we move this to package.json instead? See https://babeljs.io/docs/en/babelrc/#use-via-packagejson

Reply:
Based on research Laravel Mix needs a .babelrc file if we want to add any custom configuration.

I tried that way to put it in package.json but it didn't worked.
When we work on front-end apps without Laravel mix, maybe it works, but in this case it didn't worked.

Reply:
I see. Okay then, let's leave it as it is!